domains/Numerical.ml

```diff
@@ -255,36 +330,32 @@ module Numerical(N: NUMERICAL)(C: CONSTRAINT): PARTITION = struct
 
   (**)
 
-  let print fmt b =
```

Comment:
Sorry, my bad. I had moved it here to be able to call 'print' in some debug statements. It's back where it used to be.